PusherMainView/PusherMainView/InterfaceFactory.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+
+struct InterfaceFactory<T: NSObjectProtocol> {
+    static func create(for storyboardName: String) -> T? {
+        let aClass = T.self
+        let bundle = Bundle(for: aClass.self)
+        let storyboard = NSStoryboard(name: storyboardName, bundle: bundle)
+        let absoluteIdentifier = NSStringFromClass(aClass)
+        let module = String(String(reflecting: T.self).prefix { $0 != "." })
+        let identifier = absoluteIdentifier[module.count + 1..<absoluteIdentifier.count]
```

Comment:
@ra-malik-alayli could you give an example of string value that is used here as `absoluteIdentifier`. Is it possible maybe to use `components(separatedBy:)` String function and then access required substring using `first` or `last`?

Reply:
@imaginaris Actually I thought that module name could contain `.` but it can't. I used `components(separatedBy:)` instead.